dsl/src/commonMain/kotlin/com/freeletics/flowredux/dsl/util/AtomicInt.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 LifeTime GmbH. All rights reserved.
+ */
+
+package com.freeletics.flowredux.dsl.util
+
+public expect class AtomicInt(initialValue: Int) {
```

Comment:
How about `internal expect class`?

Reply:
💯

Reply:
Unfortunately, this class cannot be made internal due to the `typealias`. `AtomicInteger` is public and thus `AtomicCounter` has to be public, too, I'm afraid.